scripts/sysman/rasState.yml

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for RAS Get State and Clear State"
+version: "1.7"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "RAS Get State Extension Name"
+version: "1.7"
+name: $S_RAS_GET_STATE_EXT_NAME
+value: '"$XS_extension_ras_state"'
```

Comment:
I dont think XS is correct. right @wdamon-intel ?

Reply:
followed what we have in powerLimits.yml . let me know if it needs to be fixed

Reply:
@jandres742 @aravindksg XS is correct.